source/blog/2016-12-12-part2-install-origin-on-f25-atomic-host.html.md

```diff
@@ -0,0 +1,353 @@
+registry-console-1-q48xn   1/1       Running   0          9h        10.129.0.2     10.0.156.20
+router-1-nwjyj             1/1       Running   0          9h        10.0.156.20    10.0.156.20
+router-1-o6n4a             1/1       Running   0          9h        10.0.251.101   10.0.251.101
+$ 
+$ oc get svc
+NAME               CLUSTER-IP       EXTERNAL-IP   PORT(S)                   AGE
+docker-registry    172.30.2.89      <none>        5000/TCP                  9h
+kubernetes         172.30.0.1       <none>        443/TCP,53/UDP,53/TCP     9h
+registry-console   172.30.147.190   <none>        9000/TCP                  9h
+router             172.30.217.187   <none>        80/TCP,443/TCP,1936/TCP   9h
+$ 
+$ oc get routes
+NAME               HOST/PORT                                        PATH      SERVICES           PORT               TERMINATION
+docker-registry    docker-registry-default.54.204.208.138.xip.io              docker-registry    5000-tcp           passthrough
+registry-console   registry-console-default.54.204.208.138.xip.io             registry-console   registry-console   passthrough
+```
+
+_**NOTE:** If there are any pods that have failed to run you can try to
+debug with the `oc status -v`, and `oc describe pod/<podname>` commands.
+For any failed deployments you can retry with the `oc deploy <deploymentname> --retry`
+command._ 
+
+We can see that we have a pod, service, and route for both a
+`docker-registry` and a `registry-console`. The docker registry is where
+any container builds within OpenShift will be pushed and the registry
+console is a web frontend interface for the registry.
+
+Notice that there are two `router` pods and they are running on two
+different nodes; the worker nodes. We can effectively send traffic to
+either of these nodes and it will get routed appropriately. For our
+install we elected to set the `openshift_master_default_subdomain` to
+`54.204.208.138.xip.io`. With that setting we are only directing traffic
+to one of the worker nodes. Alternatively, we could have configured this
+as a hostname that was load balanced and/or performed round robin to
+either worker node.
+
+Now that we have explored the install, let's try out logging in as
+`admin` to the openshift web console at `https://54.175.0.44:8443`:
+
+![image](/images/2016-12-12-origin-install-part-2/login.jpeg)
+
+And after we've logged in, we see the list of **projects** that the
+`admin` user has access to:
+
+![image](/images/2016-12-12-origin-install-part-2/logged_in.jpeg)
+
+We then select the `default` project and can view the same applications
+that we looked at before using the `oc` command:
+
+![image](/images/2016-12-12-origin-install-part-2/logged_in_default.jpeg)
+
+At the top, there is the registry console. Let's try out accessing the
+registry console by clicking the
+`https://registry-console-default.54.204.208.138.xip.io/` link in the
+top right. Note that this is the link from the exposed route:
```

Comment:
the router should be listening on port 80 and port 443. Note that the routers run on the worker nodes so make sure that you set up your "default_subdomain" using one of those IP addresses